catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -223,6 +236,8 @@ public Metacard transform(InputStream input, String id)
             }
 
             String metacardContentType = metacard.getContentTypeName();
+            metacard.setAttribute(new AttributeImpl(Core.DATATYPE, getDatatype(metacardContentType)));
```

Comment:
The Datatype here is being set from the Tika Metadata Mime Type.  What would the PreIngestPlugin plugin look like for other InputTransformers that do not have the Mime Type available?